packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -139,7 +139,8 @@ type GetCallCompositePageFunction = ((
     unsupportedBrowserInfo?: {
       environmentInfo?: EnvironmentInfo;
       unsupportedBrowserVersionOptedIn?: boolean;
-    }
+    },
+    /* @conditional-compile-remove(call-transfer) */ transferCall?: CallState
```

Comment:
Is this line under the /* @conditional-compile-remove(unsupported-browser) */ flag?
I guess it would be not independently removable then?